src/model/labs-post.v2.yaml

```diff
@@ -4,6 +4,11 @@ type: object
 allOf:
   - $ref: ../snippets/labs-post.v1.yaml
   - properties:
+        image:
+            type: object
+            properties:
+                social:
+                    $ref: ../misc/image.v1.yaml
```

Comment:
Just to double check, the social image is optional here right?

Reply:
Because the image is required in the `labs-post` snippet and 'thumbnail` is required, `social` is not stated as required. `social` should always be considered optional. It only is included in required if `image` is optional and we need to have at least one expected property.